src/config.rs

```diff
@@ -194,6 +195,12 @@ impl ConfigBuilder {
         self
     }
 
+    /// set if you want to write colors in the logfile (default is Off)
+    pub fn set_write_log_enable_colors(&mut self, local: bool) -> &mut ConfigBuilder {
```

Comment:
Can you add a `cfg(feature = "ansi_term")` to this function? It won't do anything, if the feature is not enabled, which might be confusing to users otherwise.

Reply:
@Drakulix 
Thanks, added this change